src/layer-composer/generators/vessel-events/vessel-events.ts

```diff
@@ -106,27 +158,40 @@ const getEncounterAuthColor = (authorizationStatus: AuthorizationOptions) => {
   }
 }
 
-export const getVesselEventsGeojson = (trackEvents: Event[] | null) => {
-  if (!trackEvents) return null
-
-  return {
+export const getVesselEventsGeojson = (trackEvents: RawEvent[] | null): FeatureCollection => {
+  const featureCollection: FeatureCollection = {
     type: 'FeatureCollection',
-    features: trackEvents.map((event: Event) => ({
+    features: [],
+  }
+
+  if (!trackEvents) return featureCollection
+
+  featureCollection.features = trackEvents.map((event: RawEvent) => {
+    const authorized = event.encounter && event.encounter.authorized === true
+    const authorizationStatus = event.encounter
+      ? event.encounter.authorizationStatus
+      : ('unmatched' as AuthorizationOptions)
+
+    const lng = event.position.lng || event.position.lon || 0
```

Comment:
do you know where the difference nomenclature is coming from? I'd like to normalize and use always the same literal

Reply:
The API uses "lon", while the original carrier portal uses "lng". 
I guess I will use "lon" ang be done with it.